src/main/java/com/danielagapov/spawn/Models/Location.java

```diff
@@ -0,0 +1,27 @@
+package com.danielagapov.spawn.Models;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.io.Serializable;
+import java.util.UUID;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class Location implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
+
+    @Column(length = 200)
```

Comment:
255 is more standard (based on # bits), from what I've seen, but this is fine. Not a big deal. 

Keep in mind, at the database level, we don't need to worry about cutting off long text and such, since that's more of a responsibility on the mobile side. 

This would especially be the case if we had both a mobile & web version of the app, in that we don't want to limit the length at the database level, but let each client side make its own config.